dashboard/app/views/pd/workshop_mailer/teacher_cancel_receipt.html.haml

```diff
@@ -30,5 +30,5 @@
 - if @workshop.course == Pd::Workshop::COURSE_CSF
   %p
     Thank you,
-    %br Dave Frye
-    %br Director of Program Development and Operations, Code.org
+    %br Sam Morris
```

Comment:
What happens if someone cancels outside CSF? Do we not say thank you?

This one also seems like it should come from the workshop organizer

Reply:
lol I guess not?

Reply:
Alright well thats thing we should think about but for now lets make it the workshop organizer

Reply:
It reads a little weird here with the workshop organizer

<img width="1175" alt="Screenshot 2023-11-08 at 11 30 24 AM" src="https://github.com/code-dot-org/code-dot-org/assets/46464143/1332ce28-21e2-4517-bc3c-a1aa06427a4d">

Reply:
What if we change it to "If this is a mistake and you did not mean to cancel your workshop registration, please email oscar_organizer@example.net"